packages/create-bison-app/template/tests/e2e/helpers.ts

```diff
@@ -35,3 +70,54 @@ export async function uploadFile(page: Page, clickableTarget: Locator, filePath:
 
   await fileChooser.setFiles(filePath);
 }
+
+export async function loginAs(page: Page, userInitial: Partial<SessionUser>) {
```

Comment:
This function creates a user and logs them in by directly setting the session cookie.